File: 3oy2 Muhriddin Sobirjonov/4masala.c
#include    <stdio.h>
#include    <stdlib.h>

int main(){
    system("cls");
    int array[3][4];
    printf("12 ta son kiriting:  ");
    for(int n = 0; n < 3; n++){
        for(int m = 0; m < 4; m++){
            scanf("%d", &array[n][m]);
        }
    }
    for(int n = 0; n < 3; n++){
        for(int m = 0; m < 4; m++){
            printf("%d\t", array[n][m]);
        }
        puts("");
    }
    printf("\n 0 - qatordagi sonlar: ");
    for(int n = 0; n < 3; n++){
        for(int m = 0; m < 4; m++){
            if(n == 0){
                printf("%d\t", array[n][m]);
            }
        }
    }

    return 0;
}

File: 4oy1 Muhriddin Sobirjonov/4masala.py
import  os
os.system("cls")

a, b, c = map(int, input("3 ta son kiriting:   ").split())
yig = 0

if  a % 2 == 0:
    yig += a
if  b % 2 == 0:
    yig += b
if  c % 2 == 0:
    yig += c

print("Juft sonlar yigindisi:  ", yig)